config/config_generator/main.go

```diff
@@ -24,19 +24,21 @@ func main() {
 
 	args := flag.Args()
 	if len(args) < 1 {
-		fmt.Println("Input file is missing.")
-		os.Exit(1)
+		fmt.Println("Output file name is missing.")
+		usage()
 	}
-
-	configStore, err := config.NewFileStore("config.json", true)
-	if err != nil {
-		panic(err)
+	outputFile := args[0]
+	if f, err := os.Open(outputFile); err == nil {
+		_ = f.Close()
+		_, _ = fmt.Fprintf(os.Stderr, "File %s already exists. Not overwriting!", outputFile)
```

Comment:
There's no return here, so in fact we do go ahead and overwrite the file?

Reply:
Doh. Good catch.